README.md

```diff
@@ -1,128 +1,151 @@
-# Getting started with AWS UI Components
-
+# AWS-UI is now Cloudscape Design System
+ 
+On July 19th, 2022, we launched [Cloudscape Design System](https://cloudscape.design). Cloudscape is an evolution of AWS-UI. It consists user interface guidelines, front-end components, design resources, and development tools for building intuitive, engaging, and inclusive user experiences at scale. As part of the release, we have begun to publish npm packages to the new `@cloudscape-design` namespace.
+ 
+**We recommend migrating your existing AWS-UI applications to *[Cloudscape](https://cloudscape.design)*.**
+ 
+**Difference between AWS-UI and Cloudscape**
+Cloudscape uses a different visual theme than AWS-UI. The underlying code base remains the same.
```

Comment:
These are rendered on a single line. To prevent this and for improved semantics, I recommend to mark them as headings (##).

![image](https://user-images.githubusercontent.com/9086585/179968376-d4988794-390e-4aa9-9cec-7656ae91b06e.png)